### ISSUE TITLE
Continued XDaysOfY clean up (08/01)

### ISSUE THREAD

**Issue by anitabe404** [studydash/cards#270]
I spent some more time reading and thinking about my code. I think I've come up with some new methods and other refinements that will make the code more readable and easier to maintain. I think I just need to push forward with the changes. I don't want to get lost in my head.

**Comment by r002** [comment]
Totally concur-- I often face the same challenge!  Getting lost in one's head is totally a thing.  What helps me is keeping an explicit list of things I need to work on next, detailed out at a very granular level.

Btw, wanted to mention quickly-- congrats!  As of today, you've completed `92 total days`! 😀

![image](https://user-images.githubusercontent.com/45280066/128069054-8b35ca0e-ccac-49da-abfa-d5b0c9e010ca.png)

After today's contribution, you'll be a **_single week_** from completing your `100 days of code`!  We both started back at the beginning of May and now it's already August!  The entire summer's really flown by and we've done and learned so much! 🥳🎉

**Comment by anitabe404** [comment]
@r002 Yes, the time is going super fast. I can't believe I'm almost finished. I remember when I started that I was really nervous about actually being able to commit. Thank you for hosting this study group. It's made a huge difference for me. It's a big part of my support system.

I can't believe I only have one week left. Once I hit Day 100, I will be taking a break though.
